main/round.js

```diff
@@ -0,0 +1,60 @@
+const Frame = require('./frame')
+// Uncomment line below for node demo
+// const showBreakdown = require('./showbreakdown')
+
+class Round {
+  constructor(input = []) {
+    this.frames = input
+  }
+
+  getTotalScore() {
+    let totalScore = 0
+    this.frames.forEach(frame => {
+      totalScore += frame.score      
+    });
+    return totalScore
+  }
+
+  addRoll(points) {
+    if (this.#isFinished()) throw 'round finished'
+    this.#allocateBonusPoints(points)
+    if (!this.#hasTenFrames()) this.#fillOrCreateFrame(points)
+    
+    // uncomment the 3 lines below for node demo
+    // console.log(showBreakdown.generateTopstring(this.frames))
+    // console.log(showBreakdown.generateBottomstring(this.frames))
+    // return `The current score is: ${this.getTotalScore()}`
+  }
+
+  #fillOrCreateFrame(points) {
+    let mostRecentFrame = this.frames[this.frames.length - 1]
```

Comment:
Using a variable to access this kind of information is a great idea! Some lines in Javascript can feel very long and confusing.